lib/mercenary/presenter.rb

```diff
@@ -31,8 +31,9 @@ def options_presentation
 
     def command_options_presentation
       return nil if command.options.empty?
-
```

Comment:
This empty line is required by our style enforcer

Reply:
Fixed.